website/release_announcement_drafts/v1.3.6.md

```diff
@@ -0,0 +1,19 @@
+We're excited to announce the v1.3.6 release of JBrowse Web! Some highlights of
+this release include:
+
+## Text Searching
+
+Text searching is now available in JBrowse2!
+
+Users will now be able to create their own text indexes with the `jbrowse text-index` command and use them to search for names or identifiers via the location search box of the Linear Genome View.
+
+![searching in lgv](https://user-images.githubusercontent.com/45598764/132396329-f3fa9ed4-ace4-40b6-8eff-8289bbd3d2ca.gif)
+Searching via the location search box of the Linear Genome View
+
+For a quick introduction and a tutorial on how to add this feature to your JBrowse instance, check out this [quick start](/docs/quickstart_cli).
```

Comment:
I think this link should have a named anchor so that it links specifically to the indexing part of the page. I think it would look like this (need to check to make sure it's accurate).

```suggestion
For a quick introduction and a tutorial on how to add this feature to your JBrowse instance, check out this [quick start](/docs/quickstart_cli#indexing-feature-names-for-searching).
```

Reply:
it's accurate, committed the suggestion